tests/mmptest/src/MMPTest.cs

```diff
@@ -1,3 +1,5 @@
+// #define ENABLE_STATIC_REGISTRAR_TESTS https://bugzilla.xamarin.com/show_bug.cgi?id=58629
```

Comment:
Note for future: we can probably use `[Category ("StaticRegistrar")]` and then exclude that category when running the tests.